_includes/configuration.md

```diff
@@ -43,9 +43,6 @@ options for database configuration:
 
   - Configure the appliance to use an external PostgreSQL database
 
-#### Configuring an Internal Database
-
-{% include configuration-db.md %}
```

Comment:
This should probably stay, but we can move the `#### Configuring an Internal Database` inside the included file.  That way you can exclude it downstream if you need to.

Reply:
It is already in the includes folder.